### PR TITLE
Upgrade to image-resizer 1.0 and add default image

### PR DESCRIPTION
**Pull request by jgillman**
This updates image-resizer to `v1.0` and adds support for serving up a default image. Fixes Goldbely/tracker#2013

The [default image on Pantograph staging](https://pantograph-staging.herokuapp.com/pantograph-default.jpg) is a cat.

If you request an image that exists the image is served as normal: https://pantograph-staging.herokuapp.com/uploads/product/main_image/14814/goodes-sausage-sampler.749b0e0e2fd7aadd036f2cb87dc48142.jpg

Non-existant image images will serve a cat in the requested size: https://pantograph-staging.herokuapp.com/cfill-h100-w300/some/fake/image.jpg
## Testing locally

To test locally make sure you add a default image in your local code base and set `IMAGE_404` in your `.env` file (in Pantograph) to the location of that image relative to `LOCAL_FILE_PATH`.

> E.g. Your default image is located in `/User/jgillman/goldbely/public/defaults/cat.jpg`
> `LOCAL_FILE_PATH` is `/User/jgillman/goldbely/public`
> `IMAGE_404` would be `defaults/cat.jpg`.
## Testing remotely

Staging is set up to point at the staging Pantograph instance which should have the latest code. Missing images should be filled in: https://drydock.goldbely.com/

### CODE REVIEW

File: plugins/sources/s3.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+try {
+  // create an AWS S3 client with the config data
+  client = new s3({
+    accessKeyId: env.AWS_ACCESS_KEY_ID,
+    secretAccessKey: env.AWS_SECRET_ACCESS_KEY,
+    region: env.AWS_REGION
+  });
+  bucket = env.S3_BUCKET;
+} catch(e) {
+
```
Comment (ajsharma) [inline plugins/sources/s3.js:19]: So, if it encounters an error it silently swallows an error condition and then continues execution without indicating any error has taken place or providing alternatives?

Comment (jgillman) [inline plugins/sources/s3.js:19]: Yes, though this is the normal way to handle this: https://github.com/jimmynicol/image-resizer/blob/v1.0.0/src/streams/sources/s3.js#L19

We can make our own alterations and improvements, but I would want to do them as a separate PR.

**Comment by jgillman** [comment]
@ajsharma verbally /reviewed
